acme/acme/client.py

```diff
@@ -236,6 +220,7 @@ def _revoke(self, cert, rsn, url):
             raise errors.ClientError(
                 'Successful revocation must return HTTP OK status')
 
+
```

Comment:
nit: added whitespace

Reply:
According to PEP8, top-level functions or classes should be separated by two blank lines. This convention is followed in the other definitions in this module, it make it in sync.

Reply:
Corrected.